<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountSelectDialog</class>
 <widget class="QDialog" name="AccountSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select an Account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="msgLabel">
     <property name="text">
      <string>Select an account.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="listView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="globalDefaultCheck">
       <property name="text">
        <string>Use as default?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="instDefaultCheck">
       <property name="text">
        <string>Use as default for this instance only?</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="deselectBtn">
       <property name="text">
        <string>Deselect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
